utils/prometheus/config.go

```diff
@@ -0,0 +1,102 @@
+package prometheus
```

Comment:
Can we somehow mark in this file that it was copied from prometheus and the licencing terms this is about?
I would prefer to have a direct link to github with commit id

Reply:
I have added a comment at the package level and added a link to the origin of each structure to the comments of the structure. This may help us in the future when we need to change something